web/modules/custom/hoeringsportal_audit_log/src/Form/SettingsForm.php

```diff
@@ -80,11 +82,12 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#tree' => TRUE,
     ];
 
-    $form['logged_pages']['logged_route_names'] = [
+    $routesToLog = $config->get('routes_to_log') ? Yaml::dump($config->get('routes_to_log')) : NULL;
```

Comment:
Use the config helper here:

```suggestion
    $defaultValue = Yaml::dump(…->getRoutesToLog());
```